parachains/runtimes/assets/asset-hub-kusama/src/weights/xcm/pallet_xcm_benchmarks_fungible.rs

```diff
@@ -93,6 +93,36 @@ impl<T: frame_system::Config> WeightInfo<T> {
 			.saturating_add(T::DbWeight::get().reads(8))
 			.saturating_add(T::DbWeight::get().writes(4))
 	}
+	// Storage: `Benchmark::Override` (r:0 w:0)
+	// Proof: `Benchmark::Override` (`max_values`: None, `max_size`: None, mode: `Measured`)
+	pub fn reserve_asset_deposited() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `0`
+		//  Estimated: `0`
+		// Minimum execution time: 500_000_000_000 picoseconds.
+		Weight::from_parts(500_000_000_000, 0)
```

Comment:
It's not going to fit in a block at that size. If we want to prohibit it then we should be using max weight.

Reply:
probably thats the point, not to allow process `reserve_asset_deposited` by default,
this number was generated by:
```
let (trusted_reserve, transferable_reserve_asset) = T::TrustedReserve::get()
			.ok_or(BenchmarkError::Override(
				BenchmarkResult::from_weight(T::BlockWeights::get().max_block)
			))?;
```

and for `AssetHubs` where we allow `IsReserve` for transfer over bridge, this will have correct value in this branch: https://github.com/paritytech/cumulus/pull/2762

Reply:
@franciscoaguirre wdyt? you added `reserve_asset_deposited` benchmark :)

Reply:
before all this, 
we had there hard-coded:
```
// TODO: hardcoded - fix https://github.com/paritytech/cumulus/issues/1974
		Weight::from_parts(1_000_000_000_u64, 0)
```
which is also not good

Reply:
@gilescope @franciscoaguirre 
so the question is,
if it is better to change:
```
BenchmarkResult::from_weight(T::BlockWeights::get().max_block)
```
to:
```
BenchmarkResult::from_weight(Weight::MAX)
``

Reply:
ok, we cannot go with `Weight:MAX` now because: https://github.com/paritytech/polkadot/pull/7546#issuecomment-1650054701  because it break `pallet_xcm::reserve_transfer_assets`,
so I reverted back that hard-coded `1_000_000_000_u64`,
anyway the result of this PR is that regeneration of weights will work in master again

but as a follow-up we need to fix and finish remote weight estimation (maybe with "standard xcm weights") plus:
https://github.com/paritytech/polkadot/pull/7424
https://github.com/paritytech/polkadot/pull/7546